task-4/task-4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace task_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, user!");
+            FullEquationCalculator();
```

Comment:
Здесь уместно вывести полученное значение
```suggestion
            Console.WriteLine($"Your answer is {Math.Round(CalculateFullExpression(), 2)}");
```